python/pyspark/sql/readwriter.py

```diff
@@ -193,10 +194,22 @@ def parquet(self, *paths):
         """
         return self._df(self._jreader.parquet(_to_seq(self._sqlContext._sc, paths)))
 
+    @ignore_unicode_prefix
+    @since(1.6)
+    def text(self, path):
```

Comment:
Could be a good idea. If we do that maybe we should do it for all data sources though, and maybe not part of this?